modules/virt-configuring-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-configuring-obsolete-cpu-models_{context}"]
+= Configuring obsolete CPU models
+
+Configure a list of obsolete CPU models by editing the `HyperConverged` custom resource (CR).
```

Comment:
Is this procedure mandatory or optional? If you introduce the procedure with an imperative, that makes it sound mandatory. I am used to seeing "You can do x" for optional procedures. Your call.

Reply:
We do have a number of procedures that start with the imperative, but I get what you mean.